src/content/pages/FileUpload/ParameterForm.tsx

```diff
@@ -0,0 +1,122 @@
+﻿import React, { useEffect, useState, useRef } from 'react'
+import {
+  Box,
+  FormControlLabel,
+  IconButton,
+  Modal,
+  Switch,
+  Tab,
+  Tabs,
+  Typography,
+  TableContainer,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell
+} from '@mui/material'
+import HelpIcon from '@mui/icons-material/Help'
+import { modalStyle } from './modalStyle'
+import CancelIcon from '@mui/icons-material/Cancel'
+import ParameterModal from './ParameterModal'
+
+import parameters from '../../../../../sharedData/parameterBuckets.json'
+import { NullableBooleanInput } from 'react-admin'
```

Comment:
Since the modal was moved to its own component, there are now many unused imports here. Please remove them

Reply:
Adressed this